src/workerd/jsg/README.md

```diff
@@ -960,8 +981,14 @@ class MyFoo extends Foo {
 
 ##### `JSG_LAZY_READONLY_INSTANCE_PROPERTY(name, getter)` and `JSG_LAZY_INSTANCE_PROPERTY(name, getter)`
 
+// r: do I understand correctly that these properties are only ever read from C++ if the user doesn't
+//    set them first before reading them? making them not quite adhere to the idea of how "lazy" types
+//    generally work?
+//    so in that sense it's more like a "stealth" property, or "fallback" or "default" property? that also
+//    happens to only ever get evaluated once and then cached "client-side" (on the JS side), but that's
+//    mostly a coincidence, since the only true use case of this type wants this as an optimization anyway?
```

Comment:
"lazy" is the term used by v8 to describe these (https://v8docs.nodesource.com/node-18.2/db/df7/classv8_1_1_template.html#ab1e3f2c6a15707ef0dd782bfda690d84) so I want to stick with that terminology. Open to suggestions on how to improve the description here tho.